server/src/main/java/org/elasticsearch/cluster/NotMasterException.java

```diff
@@ -14,7 +14,13 @@
 import java.io.IOException;
 
 /**
- * Exception which indicates that an operation failed because the node stopped being the elected master.
+ * Exception indicating that an operation failed because the node stopped being the elected master.
```

Comment:
Suggest being a little more restrictive here:

```suggestion
 * Exception indicating that a cluster state update operation failed because the node stopped being the elected master.
```